src/main/java/seedu/linkedout/logic/commands/ClearCommand.java

```diff
@@ -12,12 +13,29 @@
 
     public static final String COMMAND_WORD = "clear";
     public static final String MESSAGE_SUCCESS = "LinkedOUT has been cleared!";
+    public static final String MESSAGE_FAIL = "LinkedOUT is not cleared";
+    public static final String MESSAGE_CONFIRMATION = "Are you sure you want to clear LinkedOUT app?";
+
+    private ClearAlertBox clearAlertBox = new ClearAlertBox();
 
 
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
-        model.setLinkedout(new Linkedout());
-        return new CommandResult(MESSAGE_SUCCESS);
+        if (isConfirmClear()) { // if user press yes on the confirmation box
+            clearAlertBox.showInformationAlert(ClearCommand.MESSAGE_SUCCESS);
+            model.setLinkedout(new Linkedout());
+            return new CommandResult(MESSAGE_SUCCESS);
+        } else { //if user press no on the confirmation box
+            return new CommandResult(MESSAGE_FAIL);
```

Comment:
this implementation will not work for this case since isConfirmClear() will brings up the confirmation box. So I check for else if, then confirmation box will be shown twice